### PR TITLE
Repackage prebuilt indexes for MS MARCO V2 doc

### PR DESCRIPTION
**Pull request by lintool**
adopts new, consistent schema for naming

reconfirmed regressions + integrations still working.

### CODE REVIEW

File: pyserini/prebuilt_index_info.py [hunk 1]
```diff
@@ -284,71 +284,71 @@
...
+        "filename": "lucene-inverted.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
+        "readme": "lucene-inverted.msmarco-v2-doc.20220808.4d6d2a.README.md",
         "urls": [
-            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene-index.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
+            "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene/lucene-inverted.msmarco-v2-doc-full.20220808.4d6d2a.tar.gz",
         ],
-        "md5": "fc6f546898725617eb5ca7a144bef531",
-        "size compressed (bytes)": 119537276117,
+        "md5": "66a9da92766dc7ee6eec168adf242a66",
+        "size compressed (bytes)": 119537262576,
```
Comment (sahel-sh) [inline pyserini/prebuilt_index_info.py:321]: I thought the diffs are name changes only, why is the size different?

Comment (lintool) [reply pyserini/prebuilt_index_info.py:321]: I unpacked, renamed, and repacked -> the name is part of the tarball, I think - so size changes slightly.

File: pyserini/resources/index-metadata/lucene-index.msmarco-v2-doc-d2q-t5-docvectors.20220525.30c997.README.md
Comment (sahel-sh) [inline pyserini/resources/index-metadata/lucene-index.msmarco-v2-doc-d2q-t5-docvectors.20220525.30c997.README.md:1]: Is this a generated readme? why are you removing it?

**Review by sahel-sh — APPROVED** [review]
LGTM with a couple of questions

**Pull request merged**